participation/membership-tiers.md

```diff
@@ -1,2 +1,9 @@
 |Corda Network Foundation|[Document history]({{ site.github.repository_url }}/blame/master/{{page.path}})|
 
+Currently there are two membership tiers for Corda Network:
```

Comment:
think we need to add that membership is for the legal entity.

Reply:
yes, okay